From 1e8ade54f16ddff15dbedd9686e40cab1ccac8b0 Mon Sep 17 00:00:00 2001
From: Tom Hall <tahall256@protonmail.ch>
Date: Tue, 10 Nov 2020 18:57:49 +0000
Subject: [PATCH] OpenZWave: Include parent directory

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fa5b3099d..74ed15903 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -712,6 +712,7 @@ IF(OpenZWave)
     IF(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
       include_directories(${OPENZWAVE_INCLUDE_DIRS})
     ELSE()
+      include_directories(${OPENZWAVE_INCLUDE_DIRS})
       include_directories(${OPENZWAVE_INCLUDE_DIRS}/openzwave)
     ENDIF()
     message(STATUS "OpenZWave includes found at: ${OPENZWAVE_INCLUDE_DIRS}")
-- 
2.28.0

